src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -357,6 +359,29 @@ def _store_testcase_for_regression_testing(testcase, testcase_file_path):
     # Only store crashes with bugs associated with them.
     return
 
+  if not progression_task_input.HasField('regression_testcase_url'):
+    return
+
+  regression_testcase_url = progression_task_input.regression_testcase_url
+
+  with open(testcase_file_path, 'rb') as testcase_file_handle:
+    testcase_file = testcase_file_handle.read()
+    try:
+      storage.upload_signed_url(testcase_file, regression_testcase_url)
+      logs.log('Successfully stored testcase for regression testing: ' +
+               regression_testcase_url)
+    except Exception:
```

Comment:
We shouldn't catch `Exception` I probably wouldn't bother catching: https://google.github.io/styleguide/pyguide.html

Reply:
you mean just doing: 
```
try:
   ...
except:
    logs.log_error(...)
```

Reply:
The reason I am trying to catch is to keep the log_error.